journal/src/main/java/io/camunda/zeebe/journal/file/MappedJournalSegmentReader.java

```diff
@@ -70,7 +69,6 @@ public JournalRecord next() {
   }
 
   public void reset() {
-    checkSegmentOpen();
```

Comment:
:question: What's the consequences of doing reset to a closed segment? Could it blow up in our faces?

Reply:
It does not blow up. The call to `next` or `hasNext` will return false anyway.